docker-images/jmxtrans/jmxtrans_readiness_check.sh

```diff
@@ -14,7 +14,8 @@ else
     export KAFKA_METRICS_PORT="$2"
 fi
 
-if nc -z "$KAFKA_HEADLESS_SERVICE" "$KAFKA_METRICS_PORT"; then
+nc -z "$KAFKA_HEADLESS_SERVICE" "$KAFKA_METRICS_PORT"
+if [ "$?" -ne 0 ]; then
```

Comment:
The code looks good, but there seem to be some ShellCheck comments:

```
if [ "$?" -ne 0 ]; then
     ^--^ SC2181: Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.
```

Could you have a look at it?

Reply:
Hi there, ok no problem, we have changed it to a ShellCheck compliant form

Reply:
Thanks